config/routes.rb

```diff
@@ -1,6 +1,18 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  namespace :api, defaults: {format: :json} do
+    resources :users do
+      resources :reservations 
+    end
+  end
+
+  namespace :api, defaults: {format: :json} do
+    resources :books 
+  end
```

Comment:
- [ ] When using `resources`, it's important to specify which routes to which actions you will need, because this would create a lot of unnecessary routes in your app.

Reply:
We will still improve on the project and use all the routes available.